requirements/ci.txt

```diff
@@ -8,19 +8,19 @@ backports.entry-points-selectable==1.1.0
     # via virtualenv
 distlib==0.3.3
     # via virtualenv
-filelock==3.3.1
+filelock==3.3.2
     # via
     #   tox
     #   virtualenv
-packaging==21.0
+packaging==21.2
     # via tox
 platformdirs==2.4.0
     # via virtualenv
 pluggy==1.0.0
     # via tox
 py==1.10.0
     # via tox
-pyparsing==3.0.1
+pyparsing==2.4.7
```

Comment:
`packaging==21.2` constrained the `pyparsing<3.0.0` in this [PR](https://github.com/pypa/packaging/pull/471) which caused the dowgrading of this package.